packages/block-library/src/navigation-link/link-ui.js

```diff
@@ -310,8 +310,7 @@ const LinkUITools = ( { setAddingBlock, focusAddBlockButton } ) => {
 	return (
 		<VStack className="link-ui-tools">
 			<Button
-				// TODO: Switch to `true` (40px size) if possible
-				__next40pxDefaultSize={ false }
+				__next40pxDefaultSize
```

Comment:
This can be seen, while you add navigation-link block and while you add the link.

| Before | After     |
| :-------- | :------- |
| <img width="1162" alt="navigation-link-before" src="https://github.com/user-attachments/assets/82c4f238-834b-4b17-89e1-c910422a988a"> | <img width="1162" alt="navigation-link-after" src="https://github.com/user-attachments/assets/5ad6ede4-c474-4887-a21c-b09183c2b177"> |